micropython/main.py

```diff
@@ -75,7 +76,7 @@
         pin = f.readline().decode()
 else:
     print("bootstrapping SIM " + imsi)
-    pin = bootstrap(imsi, KEY_SERVER, config["api"]["upp"])
+    pin = bootstrap(imsi, BOOT_SERVER, config["api"]["upp"], debug=config["sim"]["debug"])
```

Comment:
Why config->api->upp instead of "password"?

Reply:
same as above (laziness) :(